build.gradle

```diff
@@ -85,6 +85,9 @@ allprojects {
             java.srcDir 'src/test/java/unstable'
             compileClasspath += main.output + test.output
             runtimeClasspath += main.output + test.output
+
+            ext.guiTestResources = "${sourceSets.test.resources.getSrcDirs()[0]}/guitests"
+            ext.unstableTestResources = "${sourceSets.test.output.resourcesDir}/unstable"
```

Comment:
Source sets are assumed to be at the top level, so `output.resourcesDir` refers to `build/resources/unstableTests`, which we aren't using. We have this odd nested source set layout, where `unstableTests` is a subset of `test`.

I tried changing it to use one source set per type of test, but then that interfered with the coverage task (we'd now have to merge multiple .exec files and would have to merge them).